constants/successStory-data.ts

```diff
@@ -0,0 +1,52 @@
+import { CopyHolder } from '../types/language'
+import { SuccessStory } from '../types/successStory'
+export const copy: CopyHolder = {
+  english: {
+    title: 'Success Stories',
+  },
+
+  spanish: {
+    title: 'Historias de éxito',
+  },
+}
+export const ArturoSuccessStory: SuccessStory = {
+  english: {
+    name: 'Arturo "Cheech" Raygoza',
+
+    role: 'President of BIG E',
+    organization: 'Alan Hancock College',
+    profilePic: 'https://i.ibb.co/R0mMtMv/BIO-Picture.jpg',
```

Comment:
I don't think that we need to rely on an outside source for this. Can we host it locally or is there some prohibition against that?

Reply:
Can move into images

Reply:
@vsauceda-cfa I think that's the best idea. Go for it!